tests/FakeItEasy.Tests/ACallToTests.cs

```diff
@@ -50,5 +68,22 @@ public void CallTo_with_function_call_should_return_configuration_from_configura
             // Assert
             result.Should().BeSameAs(configuration);
         }
+
+        [SuppressMessage("Microsoft.Design", "CA1062:Validate arguments of public methods", MessageId = "0", Justification = "It's not really a public method")]

```

Comment:
> (I'm not sure why we don't declare them as local variables, by the way)

You can if you want, it's just more verbose.

Early versions of xbehave didn't allow arguments in `Scenario` methods unless they were all matched with `Example` values (just like xunit doesn't allow them in `Fact` methods, or in `Theory` methods unless they are all matched with `InlineData` values or similar).

Allowing them to be declared as method arguments instead of local variables was a feature proposed by an xbehave user. Since the feature was introduced it's become the idiomatic way of writing xbehave scenarios.